src/pages/tours/[id].tsx

```diff
@@ -73,36 +75,38 @@ const TiptapPage: NextPage = ({ propTour }: InferGetServerSidePropsType<typeof g
       Strike,
       CharacterCount,
       Underline,
-      Highlight.configure({ multicolor: true }),
-      TextAlign.configure({
-        types: ["heading", "paragraph"],
-      }),
+      Highlight.configure( { multicolor: true } ),
+      TextAlign.configure( {
+        types: [ "heading", "paragraph" ],
+      } ),
       History,
-      Image.extend({
-    //     addAttributes() {
-    //       return {
-    //         src: {
-    //           renderHTML: attributes => {
-    //             if (attributes.src) {
-    //               const src = /([a-zA-Z0-9\s_\\.\-\(\):])+$/.exec(attributes.src);
-    //               if (src) {
-    //                 const name = src[0];
-    //                 tourImages.forEach((image) => {
-    //                   if (image.name === name) {
-    //                     console.log(image.bloburl);
-    //                     return {
-    //                       src: image.bloburl,
-    //                     }
-    //                   }
-    //                 });
-                    
-    //               }
-    //             }
-    //           },
-    //         }
-    //       }
-    //     }
-      }),
+      FontFamily,
+      TextStyle,
+      Image.extend( {
+        //     addAttributes() {
+        //       return {
+        //         src: {
+        //           renderHTML: attributes => {
+        //             if (attributes.src) {
+        //               const src = /([a-zA-Z0-9\s_\\.\-\(\):])+$/.exec(attributes.src);
+        //               if (src) {
+        //                 const name = src[0];
+        //                 tourImages.forEach((image) => {
+        //                   if (image.name === name) {
+        //                     console.log(image.bloburl);
+        //                     return {
+        //                       src: image.bloburl,
+        //                     }
+        //                   }
+        //                 });
+
+        //               }
+        //             }
+        //           },
+        //         }
+        //       }
+        //     }
```

Comment:
are we holding this code for some other purpose? do we need to uncomment or stash?

Reply:
let's stash because this is Tye's code and Idk if he's planning on using it later on